web3/eth.py

```diff
@@ -868,6 +868,11 @@ def sign_munger(
         mungers=[BaseEth.call_munger]
     )
 
+    createAccessList: Method[Callable[..., Union[bytes, bytearray]]] = Method(
+        RPC.eth_createAccessList,
+        mungers=[BaseEth.call_munger]
```

Comment:
`createAccessList` doesn't take a `state_override` variable like `eth_call` does, so you'll need to create a new munger instead of using `call_munger`

Reply:
I think we can use call_munger because of this if statement:

https://github.com/ethereum/web3.py/blob/81427e69388a4a900ca1260fbb82bef91129367c/web3/eth.py#L311-L312

Reply:
But my initial thought was code reuse. That's why I used this method.